packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -636,6 +660,10 @@ contract GuardiansRegistry is ControlledRecoverable, IGuardiansRegistry, IERC900
     function _activateTokens(address _guardian, uint256 _amount, address _sender) internal {
         uint64 termId = _ensureCurrentTerm();
 
+        // Make sure the sender is allowed to activate tokens on behalf of the guardian
+        bool isAllowed = _guardian == _sender || _isActivatorAllowed(_sender) || _isActivatorAllowed(ANY_ENTITY);
```

Comment:
⚠️ I'm having second thoughts about allowing `ANY_ENTITY` here (and perhaps for locking as well).

If we allow it here, anyone can call `stakeFor()` for anyone with an existing allowance.